### ISSUE TITLE
http endpoints cleanup

### ISSUE THREAD

**Issue by jaffee** [pilosa/pilosa#1371]
### Description

We have some silly endpoints like `/slices/max` and `/info`. What does info return? Well that's kind of my point - you have no idea. What other endpoints are under `/slices`? None. Just `/slices/max`. Turns out the only info under `/info` is slicewidth. Let's roll all this information under the `/status` endpoint. This has the added benefit of putting all the information that a client library wants to know in one place!

We have a number of endpoints that are for internal use only - let's make this explicit and nest them under `/internal` then maybe people won't get angry if we change them post-1.0. I think the list is:
- `/cluster/message`
- `/fragment/*`
- `/*/attr/diff`

Imports are frame specific - why are the `/import` and `/import-value` endpoints top-level? Let's put them under the `/index/{index}/field/{field}`. Bonus - then we won't need a separate import-value because it will be clear from the field type.

This also seems like a good idea: #1283

### Success criteria (What criteria will consider this ticket closeable?)

We do all that stuff ☝️

**Comment by travisturner** [comment]
Regarding `/info`, when we discussed this the idea was to include compile-time information at a single endpoint (sliceWidth, version). See #1236.
I'm not opposed to munging everything together into a single endpoint as long as we're thoughtful about it. We just got out of a situation where `/status` returned many different things, each selectively used by different functions, so I'd like to avoid getting in that place again.
